apps/music/js/ui/views/player_view.js

```diff
@@ -763,22 +763,17 @@ var PlayerView = {
       this.audio.currentTime = Math.floor(seekTime);
     }
 
-    this.setSeekBar(0, this.audio.duration, this.audio.currentTime);
+    this.setSeekBar(this.audio.duration, this.audio.currentTime);
   },
 
-  setSeekBar: function pv_setSeekBar(startTime, endTime, currentTime) {
-    if (!isFinite(endTime)) {
-      endTime = Math.max(this.seekBar.max, currentTime);
-    }
-
+  setSeekBar: function pv_setSeekBar(endTime, currentTime) {
     if (this.seekBar.max != endTime) {
```

Comment:
In the previous code, endTime was guaranteed to be finite at this point. But now it is not. So maybe you should add the isFinite(endTime) test here, too?